eng/pipelines/apiview-review-gen-swagger.yml

```diff
@@ -1,45 +1,44 @@
-pr: none
-
-trigger: none
-
-pool:
-  name: azsdk-pool-mms-win-2022-general
-  vmImage: MMS2022
-
-variables:
-  SwaggerParserInstallPath: $(Pipeline.Workspace)/SwaggerApiParser
-
-jobs:
-- job: CreateSwaggerReviewCodeFile
-  displayName: 'Create Swagger API review token file'
-
-  variables:
-  - template: /eng/pipelines/templates/variables/globals.yml
-
-  steps:
-  - script: |
-      mkdir SwaggerApiParser
-    workingDirectory: $(Pipeline.Workspace)
-    displayName: Setup working directory for Swagger Parser.
-
-  - task: UseDotNet@2
-    displayName: 'Use .NET Core sdk 6.x'
-    inputs:
-      version: '6.x'
-
-  - script: >
-      dotnet tool install
-      Azure.Sdk.Tools.SwaggerApiParser
-      --version 1.0.0-dev.0.4
-      --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json
-      --tool-path $(SwaggerParserInstallPath)
-    workingDirectory: $(SwaggerParserInstallPath)
-    displayName: 'Install Swagger APIView Parser'
-
-  - template: /eng/pipelines/templates/steps/apiview-review-gen.yml
-    parameters:
-      Reviews: $(Reviews)
-      APIViewURL: $(APIViewURL)
-      StorageContainerUrl: $(StorageContainerUrl)
-      ApiviewGenScript: './Create-Apiview-Token-Swagger.ps1'
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-win-2022-general
+  vmImage: MMS2022
+
+variables:
+  SwaggerParserInstallPath: $(Pipeline.Workspace)/SwaggerApiParser
+
+jobs:
+- job: CreateSwaggerReviewCodeFile
+  displayName: 'Create Swagger API review token file'
+
+  variables:
+  - template: /eng/pipelines/templates/variables/globals.yml
+
+  steps:
+  - script: |
+      mkdir SwaggerApiParser
+    workingDirectory: $(Pipeline.Workspace)
+    displayName: Setup working directory for Swagger Parser.
+
+  - template: /eng/pipelines/templates/steps/install-dotnet.yml
+    parameters:
+      DotNetCoreVersion: ${{ parameters.DotNetCoreVersion }}
```

Comment:
FYI this is the actual change

Reply:
I don't see this parameter defined in this yml where is it expected to be set?

Reply:
I think you just want to use the default version here so remove passing this.

Reply:
My bad, copy-paste error. Thanks for catching this!